packages/components/src/box-control/index.js

```diff
@@ -55,7 +55,10 @@ export default function BoxControl( {
 	splitOnAxis = false,
 	allowReset = true,
 	resetValues = DEFAULT_VALUES,
+	allowNegativeValues = false,
 } ) {
+	inputProps = allowNegativeValues ? { min: -Infinity } : defaultInputProps;
```

Comment:
@JustinyAhin Won't this overwrite `inputProps`, making the prop unusable?

Reply:
@talldan in which scenario would that happen? Not sure I really understand your question

Reply:
Hmmm you have a point @talldan! Apologies, I initially missed this in my initial review.

Should probably be something like this instead:
```js
inputProps.min = allowNegativeValues ? -Infinity : defaultInputProps.min;
```

What do you think?